resources/css/style.css

```diff
@@ -223,19 +227,19 @@ textarea.materialize-textarea {
 }
 :-moz-placeholder {
   /* Firefox 18- */
-
+  text-align: center;
```

Comment:
@gkrishnan724  I think this will text centering it to be backwards compatible as its in ``` -moz-placeholder (firefox 18-)```